wire-ios-request-strategy/Sources/Payloads/Processing/PayloadProcessing+Conversation.swift

```diff
@@ -501,3 +523,15 @@ extension Payload.ConversationEvent where T == Payload.UpdateConversationConnect
     }
 
 }
+
+private extension ZMConversation {
+
+    func systemMessage(for systemMessageType: ZMSystemMessageType) -> ZMSystemMessage? {
```

Comment:
You should call this `firstSystemMessage` to be more transparent to the invoker what this method does.

Reply:
done